doc/nrf/device_guides/working_with_nrf/nrf70/gs.rst

```diff
@@ -245,3 +254,118 @@ The following samples allow you to test if the buttons (or switches) and LEDs on
+| D1               | BUCK_EN               | Enable power to nRF7002          |
++------------------+-----------------------+----------------------------------+
+| D2               | COEX_STATUS0          | Coexistence status 0             |
++------------------+-----------------------+----------------------------------+
+| D3               | COEX_REQ              | Coexistence request from host    |
++------------------+-----------------------+----------------------------------+
+| D4               | COEX_GRANT            | Coexistence grant to host        |
++------------------+-----------------------+----------------------------------+
+| D5               | SW_CTRL0              | Switch control 0                 |
++------------------+-----------------------+----------------------------------+
+| D6               | COEX_STATUS1          | Coexistence status 1             |
++------------------+-----------------------+----------------------------------+
+| D7               | HOST_IRQ              | Interrupt request to host        |
++------------------+-----------------------+----------------------------------+
+| D8               | DATA2                 | QSPI data line 2                 |
++------------------+-----------------------+----------------------------------+
+| D9               | DATA3                 | QSPI data line 3                 |
++------------------+-----------------------+----------------------------------+
+| D10              | SS                    | Slave select                     |
++------------------+-----------------------+----------------------------------+
+| D11              | MISO/DATA1            | QSPI/SPI Data line 1/ Slave Out  |
++------------------+-----------------------+----------------------------------+
+| D12              | MOSI/DATA0            | QSPI/SPI Data line 0/ Slave In   |
++------------------+-----------------------+----------------------------------+
+| D13              | CLK                   | QSPI/SPI Clock                   |
++------------------+-----------------------+----------------------------------+
+| GND              | GND                   | Ground                           |
++------------------+-----------------------+----------------------------------+
+| AREF             | N.C.                  | Not used                         |
++------------------+-----------------------+----------------------------------+
+| SDA              | N.C.                  | Not used                         |
++------------------+-----------------------+----------------------------------+
+| SCL              | N.C.                  | Not used                         |
++------------------+-----------------------+----------------------------------+
+
+Programming
+-----------
+
+To add support for the nRF7002 EK on an application running on a compatible host development board, the ``SHIELD`` setting must be specified.
+
+The nRF7002 EK features the nRF7002 companion IC.
```

Comment:
I am thinking now, this sentence could alternatively be moved in the overview section, above.
Also: do we need active links to the 7000 and 7001 product pages? Or, the common nrf70 series landing page?

Reply:
Added the sentence above the "Overview section". 

Added the link to the common nrf70 series landing page in the Reference section.